src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusHttpListenerOptions.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="PrometheusHttpListenerOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Exporter.Prometheus.HttpListener
+{
+    public class PrometheusHttpListenerOptions
+    {
+        private IReadOnlyCollection<string> httpListenerPrefixes = new string[] { "http://localhost:9464/" };
+
+        /// <summary>
+        /// Gets or sets the prefixes to use for the http listener. Default
+        /// value: http://localhost:9464/.
+        /// </summary>
+        public IReadOnlyCollection<string> HttpListenerPrefixes
```

Comment:
This'd be apt time to rename this to simply "Prefixes", as the class name is already qualifying this.

Reply:
Resolved in https://github.com/open-telemetry/opentelemetry-dotnet/pull/3430.